storagetransfer/transfer_from_aws.go

```diff
@@ -7,41 +7,42 @@ import (
 	"google.golang.org/genproto/googleapis/type/date"
 	"google.golang.org/genproto/googleapis/type/timeofday"
 	"io"
+	"os"
 	"time"
 
 	storagetransfer "cloud.google.com/go/storagetransfer/apiv1"
 	storagetransferpb "google.golang.org/genproto/googleapis/storagetransfer/v1"
 )
 
-func transfer_from_aws(w io.Writer, projectID string, jobDescription string, awsSourceBucket string, gcsSinkBucket string, startTime time.Time, awsAccessKeyID string, awsSecretKey string) (*storagetransferpb.TransferJob, error) {
+func transferFromAws(w io.Writer, projectID string, awsSourceBucket string, gcsSinkBucket string) (*storagetransferpb.TransferJob, error) {
 	// Your Google Cloud Project ID
 	// projectID := "my-project-id"
 
-	// A description of this job
-	// jobDescription = "Transfers objects from an AWS bucket to a GCS bucket"
-
 	// The name of the Aws bucket to transfer objects from
 	// awsSourceBucket := "my-source-bucket"
 
 	// The name of the GCS bucket to transfer objects to
 	// gcsSinkBucket := "my-sink-bucket"
 
-	// The time to start the transfer
-	// startTime := time.Now()
-
-	// The AWS access key credential, should be accessed via environment variable for security
-	// awsAccessKeyID := os.Getenv("AWS_ACCESS_KEY_ID")
-
-	// The AWS secret key credential, should be accessed via environment variable for security
-	// awsAccessKeyID := os.Getenv("AWS_SECRET_KEY")
-
 	ctx := context.Background()
 	client, err := storagetransfer.NewClient(ctx)
 	if err != nil {
 		return nil, fmt.Errorf("storagetransfer.NewClient: %v", err)
 	}
 	defer client.Close()
 
+	// A description of this job
```

Comment:
Comments here and following seem unnecessary, but fine to leave them if they are the same across languages.

Reply:
Yeah the comments are just copied from the argument list, I'm going to leave them to call attention to the "these are the variables you can change for your on use case!" idea, but open to changing it in the future